app_spec/test/files/links.js

```diff
@@ -77,6 +77,7 @@ module.exports = scenario => {
     t.ok(error.line)
   })
 
+// this is where the failing tests are
```

Comment:
```suggestion
```

Reply:
I think this comment can be removed now?